packages/vscode-extension/lib/wrapper.js

```diff
@@ -69,6 +69,118 @@ function useAgentListener(agent, eventName, listener, deps = []) {
   }, [agent, ...deps]);
 }
 
+function getInspectorDataForInstance(node) {
+  const renderers = Array.from(window.__REACT_DEVTOOLS_GLOBAL_HOOK__?.renderers?.values());
```

Comment:
It seems like we should get the renderer at the start of the traversal. There's no need to do all of that at every iteration of the loop

Reply:
changed to obtain the `getInspectorDataForInstance` method only once at the beginning of `traverseComponentsTreeUp`